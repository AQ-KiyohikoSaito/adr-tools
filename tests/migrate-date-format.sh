adr init
adr_file="`adr new With Old Date Format`"
sed -e 's+^Date: .*$+Date: 12/01/1992+' "$adr_file" > "$adr_file.tmp"
mv "$adr_file.tmp" "$adr_file"
adr new With Current Date Format
grep 'Date: ' doc/adr/*
adr upgrade-repository
grep 'Date: ' doc/adr/*
