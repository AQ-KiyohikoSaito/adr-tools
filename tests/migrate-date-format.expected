+ adr init
doc/adr/0001-record-architecture-decisions.md
++ adr new With Old Date Format
+ adr_file=doc/adr/0002-with-old-date-format.md
+ sed -i '' -e 's+^Date: .*$+Date: 12/01/1992+' doc/adr/0002-with-old-date-format.md
+ adr new With Current Date Format
doc/adr/0003-with-current-date-format.md
+ grep 'Date: ' doc/adr/0001-record-architecture-decisions.md doc/adr/0002-with-old-date-format.md doc/adr/0003-with-current-date-format.md
doc/adr/0001-record-architecture-decisions.md:Date: 1992-01-12
doc/adr/0002-with-old-date-format.md:Date: 12/01/1992
doc/adr/0003-with-current-date-format.md:Date: 1992-01-12
+ adr upgrade-repository
+ grep 'Date: ' doc/adr/0001-record-architecture-decisions.md doc/adr/0002-with-old-date-format.md doc/adr/0003-with-current-date-format.md
doc/adr/0001-record-architecture-decisions.md:Date: 1992-01-12
doc/adr/0002-with-old-date-format.md:Date: 1992-01-12
doc/adr/0003-with-current-date-format.md:Date: 1992-01-12
